frontend/src/components/tabs/InputOutputTab.tsx

```diff
@@ -62,8 +62,8 @@ export function InputOutputTab({ execution, namespace }: IOTabProps) {
   let inputArtifacts: ParamList = [];
   let outputArtifacts: ParamList = [];
   if (isSuccess && data) {
-    inputArtifacts = getArtifactParamList(filterEventWithInputArtifact(data));
-    outputArtifacts = getArtifactParamList(filterEventWithOutputArtifact(data));
+    inputArtifacts = getArtifactParamList(filterEventWithInputArtifact(data), '');
+    outputArtifacts = getArtifactParamList(filterEventWithOutputArtifact(data), '');
```

Comment:
These two lines shouldn't be using `''` for `artifactTypeName`. Can you take a video of `Execution node` and see if the output is matching what we expect? (`(This is an empty file by default)` should be applied to `'system.Metrics'` and `'system.ClassificationMetrics'` types artifacts)

Reply:
Video for `Execution nodes`:

https://user-images.githubusercontent.com/56132941/182493215-8b8d3a06-d0c2-4291-8d7c-bfb5ff8ff7b6.mov